test/app/languageforge/lexicon/settings/config-fields.e2e-spec.ts

```diff
@@ -717,33 +714,25 @@ describe('Lexicon E2E Configuration Fields', () => {
     });
 
     it('can open the custom field modal for an entry', async () => {
```

Comment:
@laineyhm TODO: these tests are no longer relevant.  I don't even know how they are passing as is :). Somehow you've made the tests pass, but we need need tests which exercise the add custom field dialog anymore.